### DISCUSSION TITLE
Can we make show.gauge.sh ui better!

### DISCUSSION THREAD

**Discussion by tiluckdave** [gauge-sh/tach#239] [Q&A] [answered]
Recently in #205 I saw [tach show link for Posthog](https://show.gauge.sh/?uid=b908bdbb-0cea-453d-b414-191c823dbc64). As Evan mentioned it is pretty tough to read the resulting graph and improvements are required for larger repos.

I wanted to propose making [show.gauge.sh](https://show.gauge.sh) a combination of [roadmap.sh](https://roadmap.sh) graphs and functionality similar to [whimsical.com](https://whimsical.com) Like allowing extending and collapsing dependencies of modules for better view, having different arrow styles for deprecated dependencies, different color for strict modules, etc. Utilizing canvas for this. Additional options like zoom, export to image/svg, Link to get the embed tag, etc.

**Comment by caelean** [comment]
@tiluckdave sorry for the delayed response; that'd be an awesome addition to `tach show`! I DM'ed you on Discord to chat more on implementation in the `show` repo, which is currently hosted under show.gauge.sh.

**Comment by DetachHead** [comment, marked as answer]
the ui on show.gauge.sh is no longer maintained since i took over the project. please use the either the mermaid or graphviz formats instead. [more info](https://docs.gauge.sh/usage/commands/#tach-show)
